Sources/LiveKit/Core/Room.swift

```diff
@@ -403,6 +403,24 @@ extension Room: SignalClientDelegate {
         }
         return true
     }
+
+    func signalClient(_ signalClient: SignalClient, didUnpublish localTrack: Livekit_TrackUnpublishedResponse) -> Bool {
+        log()
+
+        guard let localParticipant = localParticipant,
+              let publication = localParticipant.tracks[localTrack.trackSid] as? LocalTrackPublication else {
+            log("unknown track publication not found", .warning)
```

Comment:
maybe `unknown track publication` or `track publication not found`?

Reply:
haha yes!